src/nl_write.jl

```diff
@@ -255,4 +255,11 @@ end
 
 nl_variable(index::Integer) = "v$index"
 nl_number(value::Real) = "n$value"
-nl_operator(operator::Symbol) = "o$(func_to_nl[operator])"
+
+function nl_operator(operator::Symbol)
+    if !(operator in keys(func_to_nl))
```

Comment:
Not just `!haskey(func_to_nl, operator)`?

Reply:
Python habit where `key in dict` works.